Yoast/wordpress-seo
# Improves the redirect upsell box

## Summary

This PR can be summarized in the following changelog entry:

* Improves the redirect upsell when user clicks on create redirect in the search console overview.

## Relevant technical choices:

* I did as small refactoring, because `WPSEO_GSC` handles the hooks three different ways.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and to the build steps `grunt build:js`
* Go to the Google Search Console and press create redirect 
* See the new upsell popping up
* Go to https://github.com/Yoast/wordpress-seo-premium/pull/2183

## Quality assurance


Fixes #10979
